packages/maskbook/src/plugins/VCent/UI/TweetDialog.tsx

```diff
@@ -0,0 +1,111 @@
+import { useEffect, useState } from 'react'
+import { makeStyles, createStyles, Button } from '@material-ui/core'
+import * as TweetAPI from '../apis/index'
+import { ETHIcon } from '../Icons/ETH'
+import { VCentIcon } from '../Icons/VCent'
+import { VALUABLES_VCENT_URL } from '../constants'
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        marginTop: 20,
+    },
+
+    content: {
+        backgroundColor: '#f3f3f3',
```

Comment:
Are those colors works in both light theme and dark theme?

Reply:
i have updated this to include dark mode. this is what it looks like. I am trying to workout how to pull module '@dimensiondev/icons' and will pull updated project.
![image](https://user-images.githubusercontent.com/80292168/113319212-1e767300-9309-11eb-95e8-1a469d339d89.png)

Reply:
hi please cherry-pick the commit af9736c4a107c6eb619e1ca89f07b45780e173b6 and run `pnpm i` again then you can use `@dimensiondev/icons` package